ansible/playbooks/sap-hana-preconfigure.yaml

```diff
@@ -4,8 +4,11 @@
   become: true
   become_user : root
   vars:
-    scale_out: False
-    use_sapconf: False
+    scale_out: false
+    use_sapconf: false
+    saptune_solution: HANA
+    platform: azure
```

Comment:
Will this be configurable later?

Reply:
These are all variables, so yes.  They can be overwritten by calling ansible with the -e flag.  For example, to use sapconf instead of saptune you could do `ansible-playbook -e 'use_sapcong=true /path/to/playbook.yaml`